src/parser/powdr.lalrpop

```diff
@@ -361,7 +361,7 @@ MatchExpression: Box<Expression> = {
 }
 
 MatchArm: (Option<AbstractNumberType>, Expression) = {
-    <n:Number> "=>" <e:Expression> => (Some(n), e),
+    <n:Integer> "=>" <e:Expression> => (Some(n), e),
```

Comment:
Shouldn't this be a field element? I mean unless way say that everything is an integer inside fixed columns...

Reply:
Oh no you're right!